k8s/base/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zitadel
+  labels:
+    app: zitadel
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: zitadel
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: zitadel
+        version: v1
+    spec:
+      containers:
+      - name: zitadel
+        image: docker.pkg.github.com/caos/zitadel/zitadel:latest
+        imagePullPolicy: Always
```

Comment:
i dont think always is good, we should use `ifNotPresent`

Reply:
that is fine with me